src/app/pages/upload-files/upload-files.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="page_container">
+    <div class="page_wrapper">
+        <form class="sandbox_wrapper" [formGroup]="fileSendForm" (ngSubmit)="submit()">
+            <mat-form-field class="example-full-width" appearance="outline">
+                <mat-label>E-mail</mat-label>
+                <input formControlName="email" matInput>
+            </mat-form-field>
+            <label for="inpFile" class="btn unfilled-btn">{{ 'uploadPage.upload' | translate }}
+                <input type="file" accept=".txt, .pdf" id="inpFile" (change)="onFileChange($event)">
```

Comment:
add doc, docx formats too

Reply:
its validate on back as txt and pdf only